# Simulation settings for using moveit_sim_controllers
moveit_sim_hw_interface:
  joint_model_group: mycobot_arm
  joint_model_group_pose: init
# Settings for ros_control_boilerplate control loop
generic_hw_control_loop:
  loop_hz: 5
  # loop_hz: 300
  cycle_time_error_threshold: 0.01
# Settings for ros_control hardware interface
hardware_interface:
  joints:
    - arm1_joint
    - arm2_joint
    - arm3_joint
    - arm4_joint
    - arm5_joint
    - arm6_joint
  sim_control_mode: 1  # 0: position, 1: velocity
# Publish all joint states
# Creates the /joint_states topic necessary in ROS
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
controller_list:
  []
mycobot_trajectory_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - arm1_joint
    - arm2_joint
    - arm3_joint
    - arm4_joint
    - arm5_joint
    - arm6_joint
  gains:
    arm1_joint:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm2_joint:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm3_joint:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm4_joint:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm5_joint:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm6_joint:
      p: 100
      d: 1
      i: 1
      i_clamp: 1